storagemanager/users/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from .forms import UserRegisterForm, UserUpdateForm, ProfileUpdateForm
+from .models import Profile
+
+
+def register(request):
+    if request.method == 'POST':
+        form = UserRegisterForm(request.POST)
+        if form.is_valid():
+            # Create new user
+            form.save()
+
+            # Automatically log in new user
+            username = form.cleaned_data.get('username')
+            new_user = authenticate(username=username,
+                                    password=form.cleaned_data['password1'],
+                                    )
+            login(request, new_user)
+            messages.success(
+                request, f'{username}, your account has been created.')
+
+            # Add phone number to profile
```

Comment:
For sure :) Nothing blocks you from doing that, but if you think about just creating user in one line and then creating profile in second, remember you'd have to recreate this logic every time you create user (and, it is prone to errors, as is not enforced by any logic), at least if it makes no sense to have user without created profile. What might be easier is to create user in view and, instead of using signals, just override user model `save` method to create profile upon creation. Maybe try this approach? Or find a way to create custom user model with all the fields you need?